EnglishFilter/sections/antiadblock.txt

```diff
@@ -574,6 +574,10 @@ uaclips.com,kzclip.com,ru-clip.com,ruclip.com###morev
 !
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/80273
+||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$script,redirect=googlesyndication-adsbygoogle,important,domain=adzsafe.com
+! Incase of DNS Filtering
+adzsafe.com#%#//scriptlet("set-constant", "chk_ads_block", "noopFunc")
```

Comment:
scriptlet is enough. It works on all platforms with `$redirect` support. The rules from `content_blocker.txt` added to iOS, Safari and Content Blocker for Android(but not to AdGuard for Android)

Reply:
I don't understand. Please change it for me.

Reply:
Redirect is redundant.